apps/tests/next.config.js

```diff
@@ -38,9 +38,9 @@ const nextConfig = {
       },
     ],
   },
+  outputFileTracingRoot: path.join(process.cwd(), '../../'),
```

Comment:
it's becoming obvious that we should have a central next config that gets extended by the apps.